tests/test_weights.py

```diff
@@ -93,7 +97,7 @@ def test_weights_latitude():
     assert found.equals(expected)
 
 
-def test_weights_NaN_matching():
+def test_weights_NaN_matching():  # pylint: disable=C0103
```

Comment:
```suggestion
def test_weights_nan_matching():  # pylint: disable=C0103
```

Reply:
I was happy to make NaN special, but I'm also happy with your suggested change, so I'll go ahead with that name change.